execution/execution.py

```diff
@@ -448,3 +458,38 @@ def advisor_systems():
 
     dic_full_list['data'] = full_list
     return dic_full_list
+
+def list_stales_host():
+    INV_FILE="/tmp/inventory.json"
```

Comment:
Please, add this entry in conf.py file and use it, instead of keeping this information locally.